app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -66,13 +66,11 @@ def ems_inv_to_hashes
       # get_hosts
       get_images
       get_servers
-      get_object_store
```

Comment:
Why did you remove this? The object store stuff hasn't changed yet, shouldn't it still work?